manage.py

```diff
@@ -413,8 +542,230 @@ def test_container_key_value():
     check_container_keys_vakues()
 
 
+@manager.command
+@manager.option(
+    "-d",
+    "--datasource",
+    help="data source name, indexing all the data sources by default",  # noqa
+)
+@manager.option(
+    "-f",
+    "--folder",
+    help="data folder containing the CSV files",  # noqa
+)
+@manager.option(
+    "-r",
+    "--resource",
+    help="resource name, creating all the indexes for all the resources by default",  # noqa
+)
+@manager.option(
+    "-n",
+    "--need_convert",

```

Comment:
Can you enforce that?

Reply:
This should be optional, as the user may provide a CSV file in searchengine format.